README.md

```diff
@@ -170,6 +171,25 @@ https://demo.imgix.net/image.jpg?w=100&dpr=4&q=23 4x,
 https://demo.imgix.net/image.jpg?w=100&dpr=5&q=20 5x
 ```
 
+#### Variable Qualities
+
+By default, this library will automatically append a variable `q` parameter mapped to each `dpr` parameter when generating a [fixed-width image](#fixed-width-images) srcset.
+
+To customize variable qualities, you can pass a `variable_qualities` dictionary in the `options` while creating srcset as below:
+```py
+>>> from imgix import UrlBuilder
+>>> client = UrlBuilder("sdk-test.imgix.net")
+>>> client.create_srcset("image.jpg", {"w": 100}, {"variable_qualities": {1: 45, 2: 30, 3: 20, 4: 15, 5: 10}})
```

Comment:
For consistency, we should remove the prompts usually seen when using the python interpreter.

```suggestion
from imgix import UrlBuilder
client = UrlBuilder("sdk-test.imgix.net")
client.create_srcset("image.jpg", {"w": 100}, {"variable_qualities": {1: 45, 2: 30, 3: 20, 4: 15, 5: 10}})
```